detect.go

```diff
@@ -113,11 +117,49 @@ func detectLangInProfiles(text string, options Options, langProfileList langProf
 	}
 
 	if len(langDistances) == 0 {
```

Comment:
We should probably also check if the length of the slice is 1 to avoid the `index out of range` panic